ext/oj/fast.c

```diff
@@ -677,21 +677,23 @@ static void free_doc_cb(void *x) {
 }
 
 static void mark_leaf(Leaf leaf) {
-    switch (leaf->value_type) {
-    case COL_VAL:
-        if (NULL != leaf->elements) {
-            Leaf first = leaf->elements->next;
-            Leaf e     = first;
+    if (NULL != leaf) {
```

Comment:
Was there a code change here or just reformatting? Please use the clang formatter in any case.

Reply:
There is a code change, that initial check that `leaf` isn't null. Clang format leaves my change unaffected, but does touch quite a lot of other lines in that file.

Reply:
Odd, the .clang-format file indicates to never use tabs yet I think the diff shows the introduction of tabs unless I am mistaken.

Reply:
I think it's just showing the different in white space (adding surrounding `if (NULL != leaf` increases the indentation level).